devices/ble_hci/common-hal/_bleio/att.c

```diff
@@ -1305,24 +1272,6 @@ STATIC void process_read_type_req(uint16_t conn_handle, uint16_t mtu, uint8_t dl
     }
 }
 
-int att_read_type_req(uint16_t conn_handle, uint16_t start_handle, uint16_t end_handle, uint16_t type, uint8_t response_buffer[]) {
-    typedef struct __packed {
-        struct bt_att_hdr h;
-        struct bt_att_read_type_req r;
-    } req_t;
-
-    uint8_t req_bytes[sizeof(req_t) + sizeof(type)];
-    req_t *req = (req_t *)req_bytes;
-
-    req->h.code = BT_ATT_OP_READ_TYPE_REQ;
-    req->r.start_handle = start_handle;
-    req->r.end_handle = end_handle;
-    req->r.uuid[0] = type & 0xff;
-    req->r.uuid[1] = type >> 8;
-
-    return send_req_wait_for_rsp(conn_handle, sizeof(req_bytes), req_bytes, response_buffer);
-}
-
```

Comment:
Please don't delete this; the _bleio HCI implementation is incomplete and this may be needed later. Let's just comment it out for now.